images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -103,7 +103,8 @@ $tools.AddToolVersion("Bazelisk", $(Get-BazeliskVersion))
 $tools.AddToolVersion("Bicep", $(Get-BicepVersion))
 $tools.AddToolVersion("Buildah", $(Get-BuildahVersion))
 $tools.AddToolVersion("CMake", $(Get-CMakeVersion))
-$tools.AddToolVersion("CodeQL Action Bundle", $(Get-CodeQLBundleVersion))
+$tools.AddToolVersion("CodeQL Action Bundle (Prior Release)", $(Get-PriorCodeQLBundleVersion))
+$tools.AddToolVersion("CodeQL Action Bundle (Latest Release)", $(Get-CodeQLBundleVersion))
```

Comment:
Open to suggestions on how to name these more effectively!

Reply:
sorry no idea

Reply:
It's a little redundant but we specify the actual version number, e.g. "CodeQL Action Bundle v2.12.1" and "CodeQL Action Bundle v2.12.0".  This would be consistent with "Docker Compose v1" and "Docker Compose v2" below.

Reply:
That would look like:

`CodeQL Action Bundle v2.12.1, 2.12.1` right? I'm not sure I like the redundancy 😆 but happy to do it if you think that's clearer 👍

Reply:
All I know is that I dont't need linux on mu machine. Win 11. thats it.

Reply:
We decided to go with `CodeQL Action Bundle (Prior)` and `CodeQL Action Bundle` 😄